ui/package.json

```diff
@@ -57,7 +57,7 @@
         "@types/react-dom": "^16.8.2",
         "@types/react-form": "^2.16.1",
         "@types/react-helmet": "^6.1.6",
-        "@types/react-router-dom": "^5.3.3",
+        "@types/react-router-dom": "^4.2.3",
         "@types/superagent": "^4.1.16",
```

Comment:
curious what the issue was with dependabot updating these?

Reply:
That update broke the whole UI. Was surprised that such changes can be merged without testing..

Reply:
Wow, thanks for fixing that. I guess we need better automated tests for the UI.

Reply:
ahh, now I see all 3 of the changes you referenced: dependabot broke it, somebody reverted that, and dependabot did the same thing again afterward. I'll bring it up tomorrow when I meet with one of the leads.